packages/fuselage/src/components/Menu/V2/Menu.tsx

```diff
@@ -28,6 +28,7 @@ interface MenuButtonProps<T> extends AriaMenuProps<T>, MenuTriggerProps {
   className?: ComponentProps<typeof Box>['className'];
   pressed?: boolean;
   maxWidth?: string;
+  customButton?: React.ReactElement;
```

Comment:
`nitpick` Could it be just `button`?

Reply:
I was wondering between these options:
1. menuButton
2. button
3. customButton
4. other?

imo `customButton` describes it more accurately, but i'm ok with `button` too. 
wdyt?

Reply:
Which would be the most familiar one for someone that knows Fuselage?

Reply:
hummm, following the same reasoning as the `icon` being the option to chose a different one from the default (kebab), I agree with `button` :)